api/all/src/main/java/io/opentelemetry/api/DefaultOpenTelemetry.java

```diff
@@ -15,26 +15,25 @@
  */
 @ThreadSafe
 public final class DefaultOpenTelemetry implements OpenTelemetry {
```

Comment:
I think we can hide this class as well correct?

Reply:
Oops forgot, yeah

Reply:
Can you fix this?